modules/menu.js

```diff
@@ -0,0 +1,40 @@
+// Navigation menu
+const menu = () => {
+  const display = () => {
+    document.getElementById('diplay-list').setAttribute('class', 'non-show');
+    document.getElementById('input-data').setAttribute('class', 'show');
+    document.getElementById('contact').setAttribute('class', 'non-show');
+    document.getElementById('footer').setAttribute('class', 'show');
+  };
+  display();
```

Comment:
- [ ] Good job, I can see you used the _Luxon_ library in your app, however, the date is not displaying in the user interface (web page). Please ensure to improve upon this.

**No current date displayed on page:**
![image](https://user-images.githubusercontent.com/12745474/214111059-8174a8d7-815e-4345-8ac2-be480b4c8bbf.png)